README.md

```diff
@@ -1,2 +1,20 @@
 # ebpf-firewall
 eBPF based firewall development
+
+## Building from Source with Nix
+
+* Clone the repository:
+
+```bash 
+git clone https://github.com/tiiuae/ebpf-firewall.git
+cd ebpf-firewall
+```
+* Start nix devshell
+```bash
+./scripts/run_dev_env.sh
```

Comment:
I'm personally not big fan of wrappers like this. `nix develop` is better known and consistent with other repos.
(`--extra-experimental-features "nix-command flakes"` is not necessary when flakes are configured properly or recent NixOS like `24.05` used)

Proposal - let's guide here for `nix develop` and you can use wrappers if you prefer.